code/datums/components/simple_access.dm

```diff
@@ -0,0 +1,31 @@
+///This component allows us to give a mob access without giving them an ID card.
+/datum/component/simple_access
```

Comment:
```suggestion
/datum/component/simple_access
	dupe_mode = COMPONENT_DUPE_ALLOWED
```
There shouldn't be any outstanding bug with mobs having more than one simple access component at a time.

Reply:
Sure